hedera-mirror-rosetta/app/persistence/transaction.go

```diff
@@ -498,6 +500,15 @@ func (tr *transactionRepository) constructTransaction(sameHashTransactions []*tr
 	return result, nil
 }
 
+func getTransactionResult(transaction *transaction) interface{} {
```

Comment:
this function should move to types/constants.go, and the prototype should be
```suggestion
func GetTransactionResult(int32 code) string {
```

we know it always returns a `string`, so shouldn't use the generic empty `interface{}`